tests/scheduling_utils.py

```diff
@@ -105,7 +111,8 @@ def create_request_for_scheduler_test(
                       arrival_time=0,
                       lora_request=None,
                       pooling_params=None,
-                      cache_salt=None)
+                      cache_salt=None,
+                      block_hasher=block_hasher)
```

Comment:
Why isn't this required to be set on the requests for tests using the real scheduler and model runner?

Or the other way around- why is it required here for the mock ones?